src/ui/shared/Header.tsx

```diff
@@ -23,7 +23,7 @@ export function Header({ title, description, Icon }: HeaderProps) {
       {/* Right side: Image */}
       <div className="relative w-10 h-10 sm:w-12 sm:h-12">
         <ImageComponent
-          src="/icons/dummy_img.png"
+          src=""
```

Comment:
should there be a prop in Header that is called to populate this?

Reply:
Agree but right now I'm doubtful that we need a seperate image component to load pngs of icons and why it shouldn't just be another hero icon. I raised this in PR #39, still under consideration I believe